dmoj/cptbox/tracer.py

```diff
@@ -149,6 +150,10 @@ def __init__(
         self.protection_fault = None
 
         self._security = security
+        if security is not None:
+            self.configure_files(security.read_fs, security.write_fs)
+        else:
+            self.configure_files([], [])
```

Comment:
Is this branch necessary at all? Seems a little weird.

Reply:
Unsure. `security` defaults to None. I don't know why.

Reply:
My question was more: it seems weird that the default uninitialized state would be different from initializing with [].

Reply:
We could maybe avoid it by having a default of empty list in `_cptbox.pyx`, but I don't see how else to avoid it. `configure_files` is the function that sets the `landlock_*` attributes.

We could, in theory, have them defaulted to empty and then have this branch to set them to something, but I don't love that.